ibis/expr/datatypes/core.py

```diff
@@ -513,6 +524,11 @@ class Integer(Primitive, Numeric):
     def nbytes(self) -> int:
         """Return the number of bytes used to store values of this type."""
 
+    def castable(self, other, value: int | None = None, **kwargs) -> bool:
```

Comment:
For better clarity could you use either `target` or `to` argument name instead of other? Otherwise I need to think twice which one is the source and which one is the target.

Reply:
Happy to do that in an immediate follow up

Reply:
Actually, I'll just push up another commit

Reply:
Done in the most recent commit bb1a2c5.